app/src/test/java/com/example/benchproject/data/popular/movies/repo/PopularMoviesDataRepositoryTest.kt

```diff
@@ -52,6 +52,15 @@ internal class PopularMoviesDataRepositoryTest {
         tested.getPopularMoviesList().exceptionOrNull() shouldBe resultThrowable
     }
 
+    @Test
+    fun `when api returns throwable, object should hold exception another example`() = runTest {
+        val myNewInstanceResultThrowable = Throwable("Unable to resolve host \"api.themoviedb.org\": No address associated with hostname")
+
+        coEvery { apiService.getPopularMovies() } throws resultThrowable
+
+        tested.getPopularMoviesList().exceptionOrNull() shouldBeEqualTo myNewInstanceResultThrowable
+    }
+
```

Comment:
### The problem is here with this test case 
```
Expected <java.lang.Throwable: Unable to resolve host "api.themoviedb.org": No address associated with hostname>, actual <java.lang.Throwable: Unable to resolve host "api.themoviedb.org": No address associated with hostname>.
java.lang.AssertionError: Expected <java.lang.Throwable: Unable to resolve host "api.themoviedb.org": No address associated with hostname>, actual <java.lang.Throwable: Unable to resolve host "api.themoviedb.org": No address associated with hostname>.
```
### When I read kluent documentation I used this assertion
![image](https://user-images.githubusercontent.com/57491794/175880881-2a2ce20f-befa-411b-a8a7-fb33472c1003.png)
### And in theory, it should work